docs/source/getting-started/content/proc_troubleshooting-bundle-version-mismatch.adoc

```diff
@@ -0,0 +1,34 @@
+[id="troubleshooting-bundle-version-mismatch_{context}"]
+= Troubleshooting bundle version mismatch
+
+As {prod} releases are setup and started, instances are created with bundle information 
+and instance data.  When a new {prod} release is setup, the bundle information and instance data
+is not updated due to the customization in the previous instance data.  This will lead to errors on 
+[command]`{bin} start` command 
+
+----
```

Comment:
What's the point of this source block? Can we clarify that by adding a bit of text before the block appears?

Reply:
Isn't this explained by `This will lead to errors on [command]`{bin} start` command`?

Reply:
I've added a clarification.